packages/shared/src/server/logger.ts

```diff
@@ -0,0 +1,33 @@
+import { env } from "../env";
+import winston from "winston";
+
+const getWinstonLogger = (
+  nodeEnv: "development" | "production" | "test",
```

Comment:
@maxdeichmann We included the nodeEnv in the getLogger method in the past to decide on the logTransport to use. As it's not necessary anymore, I'd prefer to drop it. Any concerns?

Reply:
No, feel free to drop.